data/webservices.py

```diff
@@ -0,0 +1,6 @@
+from community.git import get_org_name
+
+
+def webservices_url():
```

Comment:
very silly.  this doesnt even return a URL.

See other review where you did the same thing.